### PR TITLE
[DVX-590] Check for output file existence

### PR DESCRIPTION
**Pull request by jobimrobinsantos-drizly**
Also I updated the README to be more in line with the format we use in Distributor

### CODE REVIEW

**Review by brettdugan-drizly — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -17,11 +22,12 @@ jobs:
     - uses: actions/checkout@v1
     - uses: actions/create-outputs@v0.0.0-fake
       id: test
-    - uses: LouisBrunner/checks-action@v1.1.1
+    - uses: DrizlyInc/checks-action@main
```
Comment (mikekantzer-drizly) [inline README.md:25]: can we do a versioned release instead of `main`?

**Review by mikekantzer-drizly — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jobimrobinsantos-drizly) [reply README.md:25]: Sure, but first we would need to create one

Comment (mikekantzer-drizly) [reply README.md:25]: :be the change:

**Comment by brettdugan-drizly** [comment]
Can we fix the linter?

**Comment by jobimrobinsantos-drizly** [comment]
I should have never turned on actions for this g.d. repo 😩

**Comment by jobimrobinsantos-drizly** [comment]
@brettdugan-drizly I am going to politely decline to fix the linting errors, and I'm rewinding some of my changes. The errors were not introduced by my changes, and I'm not a js/ts person (yet) anyhow.

**Comment by jobimrobinsantos-drizly** [comment]
lol it looks like I stumbled backwards into fixing the linting errors

**Review by mikekantzer-drizly — APPROVED** [review]

**Pull request merged**